specification/mongocluster/DocumentDB.MongoCluster.Management/MongoCluster.tsp

```diff
@@ -362,3 +409,56 @@ union ReplicationState {
   /** Replication link is re-configuring due to a promotion event. */
   "Reconfiguring",
 }
+
+/** Specification for a node group. */
+@removed(Versions.v2024_07_01)
+model NodeGroupSpec is NodeGroupProperties {
+  /** The node type deployed in the node group. */
+  kind?: NodeKind;
+
```

Comment:
kind should be a top level property

Reply:
Same as the previous comment - from an existing API version and if the ask isn't something that won't change the specification, then I need guidance on what 'top-level' property means in this context.

Reply:
Resolving per the other comment - this is from the last preview API version and is being removed/refactored in this initial stable API version.